File: POO/Python Files/Calculadora.py
class Calculadora:

    def __init__(self, op_1, op_2):
        self.__op_1 = op_1
        self.__op_2 = op_2

    def getOp1(self):
        return self.__op_1

    def getOp2(self):
        return self.__op_2

    def setOp1(self, op):
        self.__op_1 = op

    def setOp2(self, op):
        self.__op_2 = op

    def add(self):
        return self.__op_1 + self.__op_2

    def sub(self):
        return self.__op_1 - self.__op_2

    def mul(self):
        return self.__op_1 * self.__op_2

    def div(self):
        return self.__op_1 / self.__op_2

c = Calculadora(8.0, 2.0)
c.setOp1(10.0)
c.setOp2(5.0)
print("OP1: %f -- OP2: %f" %(c.getOp1(), c.getOp2()))
print("Add: %f --- Sub: %f --- Mul: %f --- Div: %f\n" %(c.add(), c.sub(), c.mul(), c.div()))

File: POO/Java Files/ListaEx1/ex2/Calculadora.java
public class Calculadora {

    private double op_1;
    private double op_2;

    public double getOp1() { return op_1; }
    public double getOp2() { return op_2; }

    public void setOp1(double op) { this.op_1 = op; }
    public void setOp2(double op) { this.op_2 = op; }

    public double add() { return this.op_1 + this.op_2; }
    public double sub() { return this.op_1 - this.op_2; }

    public static void main(String[] args) {
        Calculadora c = new Calculadora();
        c.setOp1(10.0);
        c.setOp2(20.0);
        System.out.println(c.add());
    }
}
